src/azul/logging.py

```diff
@@ -133,6 +136,22 @@ def silent_es_log_level():
 
 
 es_log = logging.getLogger('elasticsearch')
+aws_log = logging.getLogger('aws')
```

Comment:
This should live in `deployment.py`. The only reason we have a variable for the `es` logger here in this module is because the module owning that logger is not our code, but in a dependency, and because we adjust that loggers log level here in this module, globally and for tests (`silenced_es_logger`). We don't need any of that for the `aws` logger.